gapic/templates/noxfile.py.j2

```diff
@@ -6,7 +6,7 @@ import os
 import nox  # type: ignore
 
 
-@nox.session(python=['3.6', '3.7'])
+@nox.session(python=['3.6', '3.7', '3.8'])
```

Comment:
Could you also add 3.9?
```suggestion
@nox.session(python=['3.6', '3.7', '3.8', '3.9'])
```

Reply:
I thought the 3.9 requirement was optional for now since it's newish.

Reply:
I've already added 3.9 to the noxfile template the cloud client libraries use. (https://github.com/googleapis/synthtool/blob/d5fc0bcf9ea9789c5b0e3154a9e3b29e5cea6116/synthtool/gcp/common.py#L221) It's my understanding that while there are a few packages missing wheels the generated unit tests should all be runnable with 3.9.